Arcs/Arcs.cpp

```diff
@@ -1077,49 +1966,97 @@ int main(int argc, char** argv)
             exit(EXIT_FAILURE);
         }
     }
+    
+    /* Check if user specified an option contradicting with the method choosen. */
+    if((params.arks && arcsOnly) || (!params.arks && arksOnly)){
+        cerr << PROGRAM ": error: You specified an option that does not match with method choosen.\nCheck --help for method specific options.\n";
+        die = true;
+    }
 
-    // Set base name if not previously set.
```

Comment:
Why did you decide to delete this code? Is it actually just moved?

Reply:
Yes it is just moved.